docs/v2-docs-guide.md

```diff
@@ -128,15 +247,15 @@ function Example() {
 <MultiStepTabItem value="js">
 
 ```jsx
-import React, { useState } from "react";
- 
+import React, { useState } from 'react';
+
 function Example() {
   const [count, setCount] = useState(0);
-   
+
   return (
     <div>
       <p>You clicked {count} times</p>
-      <button onClick={() => setCount(count + 1)}>
+      <button className="button primary" onClick={() => setCount(count + 1)}>
```

Comment:
Added this to bring the text wider than the container. We need to fix this to scroll horizontally.

Reply:
@jjwill It was scrolling horizontally but I figured it would be better to have it wrap. For smaller screens I think it's really a pain to have to scroll for lines

Reply:
We will need this to scroll